ManatiCR/artavia
# AR-23 team persona #1

[AR-23](https://manati.atlassian.net/browse/AR-23)
[Figma designs](https://www.figma.com/file/sRoXqR6hyiNvc8moaFeZA6/Artavia-Consultor%C3%ADa)
-
**Steps to test**
- Run `sudo rm -r web && rm composer.lock && composer install && ./scripts/artavia_local_install.sh`
- Create a node persona
- It should have all the fields specified in the ticket
- You should be able to create taxonomy terms and clientes from the persona create form
